File: AplicacionPeliculas/inicio.vb
Public Class inicio


    Private Sub Button2_Click(sender As Object, e As EventArgs) Handles Button2.Click
        Pelicula.Show()
    End Sub

    Private Sub Button3_Click(sender As Object, e As EventArgs) Handles Button3.Click
        Genero.Show()
    End Sub

    Private Sub Button4_Click(sender As Object, e As EventArgs) Handles Button4.Click
        Dim query As String = "SELECT idPais, Pais  FROM Pais where estatus = 1"
        Dim dt As DataTable = Conecction.SelectQuery(query)
        BusquedaPais.DataBuscaPais.DataSource = dt
        BusquedaPais.Show()
    End Sub

    Private Sub Button5_Click(sender As Object, e As EventArgs) Handles Button5.Click
        Dim query As String = "SELECT id, nombre  FROM Genero where estatus = 1"
        Dim dt As DataTable = Conecction.SelectQuery(query)
        BuscaGenero.DataBuscagenero.DataSource = dt
        BuscaGenero.Show()
    End Sub

    Private Sub Button6_Click(sender As Object, e As EventArgs) Handles Button6.Click
        Dim query As String = "SELECT idCompania, nombre  FROM Compania where estatus = 1"
        Dim dt As DataTable = Conecction.SelectQuery(query)
        BuscaCompania.DataBuscaCompania.DataSource = dt
        BuscaCompania.Show()
    End Sub

    Private Sub Button7_Click(sender As Object, e As EventArgs) Handles Button7.Click
        Dim query As String = "SELECT idIdioma, Idioma  FROM Idioma where estatus = 1"
        Dim dt As DataTable = Conecction.SelectQuery(query)
        BuscaIdioma.dataBuscaIdioma.DataSource = dt
        BuscaIdioma.Show()
    End Sub

    Private Sub Button8_Click(sender As Object, e As EventArgs) Handles Button8.Click
        Idioma.Show()
    End Sub

    Private Sub Button9_Click(sender As Object, e As EventArgs) Handles Button9.Click
        Pais.Show()
    End Sub

    Private Sub Button10_Click(sender As Object, e As EventArgs) Handles Button10.Click
        Compañia.Show()
    End Sub



    Private Sub inicio_Load(sender As Object, e As EventArgs) Handles MyBase.Load
        Label1.BackColor = Color.Transparent
    End Sub

    Private Sub PictureBox1_Click(sender As Object, e As EventArgs) Handles PictureBox1.Click
        Usuario.Show()
    End Sub

    Private Sub PictureBox2_Click(sender As Object, e As EventArgs) Handles PictureBox2.Click
        Peliculas.Show()
    End Sub


End Class

File: AplicacionPeliculas/inicio.cs
using System;
using System.Drawing;

namespace AplicacionPeliculas
{
    public partial class inicio
    {
        public inicio()
        {
            InitializeComponent();
        }


        private void Button2_Click(object sender, EventArgs e)
        {
            My.MyProject.Forms.Pelicula.Show();
        }

        private void Button3_Click(object sender, EventArgs e)
        {
            My.MyProject.Forms.Genero.Show();
        }

        private void Button4_Click(object sender, EventArgs e)
        {
            string query = "SELECT idPais, Pais  FROM Pais where estatus = 1";
            var dt = Conecction.SelectQuery(query);
            My.MyProject.Forms.BusquedaPais.DataBuscaPais.DataSource = dt;
            My.MyProject.Forms.BusquedaPais.Show();
        }

        private void Button5_Click(object sender, EventArgs e)
        {
            string query = "SELECT id, nombre  FROM Genero where estatus = 1";
            var dt = Conecction.SelectQuery(query);
            My.MyProject.Forms.BuscaGenero.DataBuscagenero.DataSource = dt;
            My.MyProject.Forms.BuscaGenero.Show();
        }

        private void Button6_Click(object sender, EventArgs e)
        {
            string query = "SELECT idCompania, nombre  FROM Compania where estatus = 1";
            var dt = Conecction.SelectQuery(query);
            My.MyProject.Forms.BuscaCompania.DataBuscaCompania.DataSource = dt;
            My.MyProject.Forms.BuscaCompania.Show();
        }

        private void Button7_Click(object sender, EventArgs e)
        {
            string query = "SELECT idIdioma, Idioma  FROM Idioma where estatus = 1";
            var dt = Conecction.SelectQuery(query);
            My.MyProject.Forms.BuscaIdioma.dataBuscaIdioma.DataSource = dt;
            My.MyProject.Forms.BuscaIdioma.Show();
        }

        private void Button8_Click(object sender, EventArgs e)
        {
            My.MyProject.Forms.Idioma.Show();
        }

        private void Button9_Click(object sender, EventArgs e)
        {
            My.MyProject.Forms.Pais.Show();
        }

        private void Button10_Click(object sender, EventArgs e)
        {
            My.MyProject.Forms.Compañia.Show();
        }



        private void inicio_Load(object sender, EventArgs e)
        {
            Label1.BackColor = Color.Transparent;
        }

        private void PictureBox1_Click(object sender, EventArgs e)
        {
            My.MyProject.Forms.Usuario.Show();
        }

        private void PictureBox2_Click(object sender, EventArgs e)
        {
            My.MyProject.Forms.Peliculas.Show();
        }


    }
}